addons/stock/tests/test_old_rules.py

```diff
@@ -593,7 +593,19 @@ def test_pick_ship_1(self):
             'warehouse_id': self.warehouse_2_steps.id,
             'group_id': procurement_group2.id,
             'origin': 'origin2',
-        })
+        },
+        {
+            'name': 'third out move',
+            'procure_method': 'make_to_order',
+            'location_id': ship_location.id,
+            'location_dest_id': customer_location.id,
+            'product_id': self.productB.id,
+            'product_uom': self.uom_unit.id,
+            'product_uom_qty': 1.0,
+            'warehouse_id': self.warehouse_2_steps.id,
+            'group_id': procurement_group2.id,
+            'origin': 'origin2',
```

Comment:
I think it is even more interesting to check that the order is not altered by setting these values to these of origin1:
```suggestion
            'group_id': procurement_group1.id,
            'origin': 'origin1',
```